recipes/azure-cli-core/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "azure-cli-core" %}
+{% set version = "2.0.56" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 95890315988be0db1405e7f2850b52f5cb5f3a04a9779995624252e9ba8996fe
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - adal >=1.2.0
+    - antlr-python-runtime
+    - argcomplete >=1.8.0
+    - azure-cli-telemetry
+    - azure-mgmt-resource ==2.1.0
+    - colorama >=0.3.9
+    - humanfriendly >=4.7
+    - jmespath
+    - knack ==0.5.1
+    - msrest >=0.4.4
+    - msrestazure >=0.4.25
+    - paramiko >=2.0.8
+    - pip
```

Comment:
Is `pip` really a runtime dependency? (It's fine if it is, just unusual)

Reply:
It is listed in package's setup.py, so looks like it